packages/quantic/force-app/main/default/lwc/quanticResultQuickview/e2e/pageObject.ts

```diff
@@ -0,0 +1,38 @@
+import type {Locator, Page, Request} from '@playwright/test';
+import {isUaClickEvent} from '../../../../../../playwright/utils/requests';
+
+export class ResultQuickviewObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get quickviewButton(): Locator {
+    return this.page.locator('c-quantic-result-quickview button');
```

Comment:
If possible please try with a `getByRole` as much as possible for buttons selectors.
This also helps us with accessibility concerns because the `getByRole` also represents some of the pre-requisites for the buttons to be picked up correctly by screen readers for example.

(But sometimes we can't get it to work in Salesforce, so if it doesn't work, just leave as-is and reply to this comment)